content/geoip/docs/databases/enterprise.md

```diff
@@ -0,0 +1,256 @@
+---
```

Comment:
Should we delete the corresponding  `.mdx` file? Then I think we can also delete the following components:

```
import BinaryDatabases from './_binary-databases.mdx';
import ConversionUtility from './_conversion-utility.mdx';
import CsvDatabasesIntro from './_csv-databases-intro.mdx';
import DataPrivacyWarning from './_data-privacy-warning.mdx';
import ExampleFiles from './_example-files.mdx';
import FileFormat from './_file-format.mdx';
import APIs from './_apis.mdx';
```

Reply:
Is it okay to just delete the components at the end of the migration?  Leaving them around isn't hurting anything and grepping for usage is easily forgotten when making PRs.

Reply:
Yeah that's fine. With similar work (i.e. Go to Perl pages) I think it was the standard to remove old code that isn't being used anymore, but I can see how this is different.

From a review perspective though it is slightly easier to confirm that the corresponding code from the `.mdx` file was copied over when the deleted component / files are included in the same PR rather than opening a new window to find the respective file. Not a deal breaker though.